Capri.Database/Entities/Promoter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Capri.Database.Entities.Identity;
+
+namespace Capri.Database.Entities
+{
+    public class Promoter : IEntity
+    {
+        [Key]
+        public Guid Id { get; set; }
+        public virtual ICollection<Proposal> Proposals { get; set; }
+        [Required]
+        public bool CanSubmitBachelorProposals { get; set; }
```

Comment:
Also we talked about submitting masters proposals (just for extending our application flexibility).